improver/calibration/rainforest_calibration.py

```diff
@@ -465,14 +479,16 @@ def _calculate_error_probabilities(
         """
         error_probability_cube = self._prepare_error_probability_cube(forecast_cube)
 
-        input_dataset, feature_variables = self._prepare_features_array(feature_cubes)
+        input_dataset = self._prepare_features_array(feature_cubes)
 
-        forecast_variable = forecast_cube.name()
-        precipitation_ind = feature_variables.index(forecast_variable)
-        precip_forecast = input_dataset[:, precipitation_ind]
+        forecast = forecast_cube.data.ravel()
```

Comment:
I think `forecast_data` might be a better choice of name here. This will make the distinction from `forecast_cube` more obvious.

Reply:
Fixed